### ISSUE TITLE
Add option to set enemy/team color instead of Forbidden/Icy

### ISSUE THREAD

**Issue by bonkmaykrQ** [TeamForbiddenLLC/warfork-qfusion#170]
In CPMA/Qk Live you can set a color that automatically applies to enemies or teammates, but Warfork instead only lets you set a color for each team regardless of which one you are playing on.

So if you switch teams, you suddenly have to remember which color is actually your enemy, and on top of that, this poses a visibility problem because many of us use forced player colors to make enemies easier to spot, but we can't set the same color for each team or else we won't be able to tell the teams apart from one another. If we make friendlies a less vibrant color to make them stick out less and focus on opponents only, then that has to be switched around every time the player switches teams.

tl:dr; the current "force model color" options are not great and aren't relative to whatever team the player is actually on, each team has a fixed color at all times and this might be a problem

**Comment by learn-more** [comment]
That is exactly why the option 'Force your team as FORBIDDEN' exists.

**Comment by bonkmaykrQ** [comment]
> That is exactly why the option 'Force your team as FORBIDDEN' exists.
that exists?

**Comment by Gelmo** [comment]
> > That is exactly why the option 'Force your team as FORBIDDEN' exists.
> 
> that exists?

Yes but it's not obvious to someone that isn't looking. I agree that things should be changed to team/enemies
